templates/cicd/pipeline_cfn.yml

```diff
@@ -200,11 +200,13 @@ Resources:
           - Effect: Allow
             Action:
               - s3:PutObject
-              - s3:PutObjectAcl
               - s3:GetBucketPolicy
               - s3:GetObject
-              - s3:GetObjectAcl
               - s3:ListBucket
+              {{- if eq .Source.ProviderName "Bitbucket"}}
```

Comment:
I wonder what we can do here so that the template doesn't need to be touched for other sources that require a codestar connection. Just food for thought, we don't need to update it in this PR!